edge-modules/api-proxy-module/azure-iot-mqtt/README.md

```diff
@@ -0,0 +1,34 @@
+An Azure IoT client library. It implements [the Azure IoT Hub MQTT protocol.](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-mqtt-support)
+
+
+# Features
+
+- Device client
+	- Receive initial twin state and updates
+	- Receive and respond to direct method requests
+
+- Module client
+	- Receive and respond to direct method requests
```

Comment:
receive initial twin state and updates like what in device client, right?

Reply:
This md file is actually the SDK from Arnav but yes.